packages/react/src/ErrorBoundary.en.mdx

```diff
@@ -48,27 +48,33 @@ const ErrorAfter4s = () => {
 
 ![Example banner](/gif/errorboundary-example.gif)
 
-#### Importing External Components
+:::tip
+
+## Define component as `<ErrorBoundary/>`'s fallback
+
+### ErrorBoundaryFallbackProps
 
-If you want to deliver a declared component to fallback, you can use the `ErrorBoundaryFallbackProps` type to declare the component.
+If you want to deliver a declared component as `<ErrorBoundary/>`'s fallback, you can use the `ErrorBoundaryFallbackProps` type to declare the component easily.
 
 ```tsx
-import { ErrorBoundaryFallbackProps } from '@suspensive/react'
+import type { ErrorBoundaryFallbackProps } from '@suspensive/react'
 
-const Fallback = ({ reset, error }: ErrorBoundaryFallbackProps) => (
+const ErrorBoundaryFallback = ({ reset, error }: ErrorBoundaryFallbackProps) => (
```

Comment:
Because we have `<Suspense/>`'s fallback, I renamed it as ErrorBoundaryFallback to express it as `<ErrorBoundary/>`'s fallback, not `<Suspense/>`'s one definitely